scripts/signature.py

```diff
@@ -93,7 +102,100 @@ def diff(args):
         """
         )
 
-    return exit_status
+    return delta, exit_status
+
+
+def diff(args):
+    """Diffs Velox function signatures."""
+    return diff_signatures(args)[1]
+
+
+def bias(args):
+    """Biases a provided fuzzer with newly added functions."""
+
+    delta, status = diff_signatures(args)
+
+    # Return if the signature check call flags incompatible changes.
+    if status:
+        return status
+
+    if not len(delta):
+        print(f"{bcolors.BOLD} No changes detected: Nothing to do!")
+        return 0
+
+    function_set = set()
+    for items in delta.values():
+        for item in items:
+            function_set.add(item.get_root_key())
+
+    # Some functions are prefixed with presto_ , spark_
+    function_dict = {}
+    for item in function_set:
+        split = item.split("_")
+        if len(split) > 1:
+            function_dict.setdefault(split[0], []).append(split[1])
+        else:
+            function_dict.setdefault("presto", []).append(split[0])
+
+    print(f"{bcolors.BOLD}Functions to be biased: {function_dict}")
+
+    for engine, functions in function_dict.items():
+        fuzzer_output = f"/tmp/fuzzer_{engine}.log"
+        if engine == "presto":
+            command = f"""./_build/debug/velox/expression/tests/velox_expression_fuzzer_test \
```

Comment:
Do we need to call this external process from here? Could we, for example, have this python script to just generate the `assign_function_tickets` string, then writing this to a file and simply using this from bash?   

So the API from circleCI could be something like
```
$ signature.py --bias --base /tmp/base --contender /tmp/contender --output /tmp/function_tickets
$ ./velox_expression_fuzzer_test --assign_function_tickets $(cat /tmp/function_tickets) ...
```

If possible, we should try to avoid repeating this long command line and centralize their call sites directly in the circleCI script (or in the Makefile)

Reply:
Yep that makes more sense.